starboard/raspi/shared/install_target.gni

```diff
@@ -52,6 +52,13 @@ template("install_target") {
         rebase_path(outputs[0], root_build_dir),
         rebase_path(inputs[0], root_build_dir),
       ]
+
+      if (current_toolchain == starboard_toolchain &&
+          invoker.type == "executable" && sb_is_modular && !sb_is_evergreen) {
```

Comment:
Why is `!sb_is_evergreen` needed here?

In `starboard/build/config/BUILDCONFIG.gn` this lived in the loader target which is only evergreen. Not sure I understand exactly why it's moving here. Then again, I didn't fully understand why this dep was added in the first place.